languages/dart/keywords/abstract.md

```diff
@@ -0,0 +1,110 @@
+# Abstract
+
+The keyword `abstract` is used to define an *abstract class* - a [class][reference-class] that can not be instantiated 
+directly, which means you can not create an [object][reference-object] from it. Conversely, a class that can be 
+instantiated directly is called a *concrete class*.
+
+Dart does not have [interfaces][reference-interface], a type that hides implementation details and show only 
+functionality to the user. They let you focus on **what** the object does instead of **how** it does it.
+
+Instead Dart has abstract types which can be used as interfaces, but do allow any and all methods to be implemented. So 
+it's up to the user how much implementation, if any, is in the abstract class.
+
+```dart
+// This class is declared abstract and thus can't be instantiated.
+abstract class CoffeeMachine {
+  //Body of abstract class
+}
+
+// Abstract classes can't be instantiated.
+CoffeeMachine coffeeMachine = CoffeeMachine();
+```
+
+An abstract class can have constructors (including [factories][reference-factory]), [fields][reference-field], 
+[methods][reference-method] as well as [static][reference-static] [member][reference-field].
```

Comment:
```suggestion
[methods][reference-method] as well as [static][reference-static] [member][reference-member].
```
`member` refers to field/setters/getters/methods that are accessible via an instance object.

Reply:
The reference link I believe should be singular `reference-member` since the page will be talking about what is a member as a concept.